webapp/src/views/ServiceList.vue

```diff
@@ -1,21 +1,32 @@
 <template>
   <div class="container">
     <h1>Dienstzuteilung</h1>
+    <multiselect
+      v-model="selectedServices"
+      placeholder="Bitte Auswählen"
+      label="name"
+      track-by="id"
+      :options="services"
+      :multiple="true"
+      @select="fetchServiceHistory"
+      @remove="fetchServiceHistory"
+      v-if="services != null && services?.length > 0"
+    ></multiselect>
 
-    <select
-      class="form-select"
-      aria-label="Default select example"
-      @change="fetchServiceHistory(($event.target as HTMLSelectElement).value)"
-    >
-      <option disabled selected>Bitte auswählen</option>
-      <option
-        v-for="(service, index) in services"
-        :key="index"
-        :value="service.id"
-      >
-        {{ service.name }}
-      </option>
-    </select>
+    <div>
+      <label class="typo__label">Sortieren nach:</label>
+      <multiselect
```

Comment:
Why did you make this a `<multiselect>`? In my opinion an HTML `<select>`  tag would be more appropriate as we want to select exactly one sorting way.